### PR TITLE
build: bump ti plugins version and enable validate config job

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

**Review by hi-rustin — APPROVED** [review]
Thanks!

File: prow/jobs/ti-community-infra/configs/configs-presubmits.yaml [hunk 1]
```diff
@@ -14,6 +14,18 @@ presubmits:
               - --plugin-config=prow/config/plugins.yaml
               - --config-path=prow/config/config.yaml
               - --job-config-path=prow/jobs
+    - name: pull-tichi-validate-external-plugin-yaml
```
Comment (hi-rustin) [inline prow/jobs/ti-community-infra/configs/configs-presubmits.yaml:17]: It's should be pull-configs-validate-external-plugin-yaml.

**Review by hi-rustin — CHANGES_REQUESTED** [review]

**Comment by hi-rustin** [comment]
@Mini256  Could you please test it on this PR?

**Comment by Mini256** [comment]
It’s strange why the validate config job is not started, is there any problem here? @hi-rustin

**Comment by hi-rustin** [comment]
This centrally configured job cannot be tested in the PR where the job is submitted.
 
 We can just merge it and test it. Please revert external plugins config.

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**